File: nvidia_backup_20260629_131053/dump_db.py
import sqlite3

db_path = '/root/wrapper/nvidia/metrics.db'
conn = sqlite3.connect(db_path)
conn.row_factory = sqlite3.Row
cur = conn.cursor()

print("--- TABLES ---")
cur.execute("SELECT name FROM sqlite_master WHERE type='table'")
tables = [row[0] for row in cur.fetchall()]
print(tables)

for table in tables:
    print(f"\n--- TABLE: {table} ---")
    cur.execute(f"PRAGMA table_info({table})")
    info = cur.fetchall()
    print("Schema:", [f"{i['name']} ({i['type']})" for i in info])
    
    cur.execute(f"SELECT * FROM {table} ORDER BY rowid DESC LIMIT 10")
    rows = cur.fetchall()
    for row in rows:
        print(dict(row))

conn.close()

File: nvidia_backup_20260629_131053/dump_db.js
const sqlite3 = require('sqlite3').verbose();
const db = new sqlite3.Database('/root/.hermes/profiles/ilma/vane/data/db.sqlite');

db.all("SELECT name FROM sqlite_master WHERE type='table'", [], (err, rows) => {
  if (err) {
    console.error(err);
    return;
  }
  console.log("Tables:", rows);
  
  // Dump configurations table
  db.all("SELECT * FROM sqlite_master WHERE type='table'", [], (err2, schemas) => {
    schemas.forEach(s => {
      console.log(`\nTable Schema: ${s.name}`);
      console.log(s.sql);
    });
    
    // Dump actual settings if they exist
    db.all("SELECT * FROM model_providers", [], (err3, providers) => {
      if (!err3) {
        console.log("\nProviders:", providers);
      }
      db.all("SELECT * FROM models", [], (err4, models) => {
        if (!err4) {
          console.log("\nModels:", models);
        }
        db.close();
      });
    });
  });
});
